docs/webassembly.md

```diff
@@ -0,0 +1,19 @@
+# WebAssembly
+
+## Quick start
+
+1. Compile go code to wasm
+
+```bash
+cd wasm
+GOOS=js GOARCH=wasm go build -o ./assets/jwt.wasm .
+```
+
+1. Run go web server
+
+```bash
+cd server
+go run main.go
+```
```

Comment:
**Suggestion:** Could you add these tasks to the Makefile?

Reply:
added make target